elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/ElasticsearchProjectionConstants.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.backend.elasticsearch;
+
+/**
+ * Projection constants specific to the Elasticsearch backend.
+ *
```

Comment:
@gsmet, I commented on the issue; I'm wondering how that'd play together with projecting actual fields? Right now I feel we don't gain much for the price of more verbose usage.